spec/tapioca/compilers/dsl/action_mailer_spec.rb

```diff
@@ -0,0 +1,85 @@
+# typed: false
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe(Tapioca::Compilers::Dsl::ActionMailer) do
+  describe("#initialize") do
+    it("gathers no constants if there are no ActionMailer subclasses") do
+      expect(subject.processable_constants).to(be_empty)
+    end
+
+    it("gathers only ActionMailer subclasses") do
+      content = <<~RUBY
+        class NotifierMailer < ActionMailer::Base
+        end
+
+        class User
+        end
+      RUBY
+
+      with_content(content) do
+        expect(subject.processable_constants).to(eq(Set.new([NotifierMailer])))
```

Comment:
Can this also please ensure that subclasses of `NotifierMailer` are also considered properly?